e2e/tests/admin.spec.ts

```diff
@@ -100,19 +100,24 @@ test('manage users', async ({ page, userToAdd }) => {
   // Then I should see the 'User deleted' banner
   await userListPage.shouldShowUserDeletedBanner()
 
-  // Given a user is not visisble in the list
-  const userToSearchFor = 'AutomatedTestUser'
-  await userListPage.shouldNotShowUser(userToSearchFor)
-
-  // When I search for them
-  await userListPage.search(userToSearchFor)
-
-  // Then I should be able to see their name
-  await userListPage.shouldShowUser(userToSearchFor)
-
-  // When I click on their name
-  await userListPage.clickEditUser(userToSearchFor)
-
-  // Then I should be taken to the Edit User page
-  await editUserPage.shouldShowUserName(userToSearchFor)
+  // The following behaviour only exists in CI
+  // Locally users are hard deleted. Once they are deleted they cannot be found via search as they are removed from the DB and only reseeded when the local API is restarted
+  // In deployed envs the user is soft deleted and can be found via search
+  if (process.env.CI) {
```

Comment:
I agree that we don't need to fix this now -- but I wonder if this is dangerous pattern:

```
 if (process.env.CI) {
  some test which may well fail in CI and not be possible to reproduce locally
}
```

Would it be better to extract this to a separate test which we [skip](https://playwright.dev/docs/api/class-test#test-skip) for now? That way it would perhaps be more obvious that it's a piece of work which we need to do, rather than an exception to our rule that the tests should run locally?

Reply:
Yeah that makes sense to me.  I agree this is a dangerous pattern.
When originally adding this test I wanted to highlight this behaviour as it seems a bit strange but I'm not sure it's desirable to change the behaviour of the deployed code (as users may want to find previously deleted users) OR the code running locally (I think it may be difficult to recreate the live-like soft-delete).